doofinder-for-woocommerce/includes/class-thumbnail.php

```diff
@@ -74,7 +74,8 @@ public function get()
 	 * @param string $thumbnail_id Thumb ID por check
 	 * @return string URL of Thumb or false
 	 */
-	private function get_image_url_by_size($thumbnail_id){
+	private function get_image_url_by_size($thumbnail_id)

```

Comment:
I think this is caused by automatic formatting, but this new format goes against WP PHP coding standards: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#brace-style

Reply:
I'm sorry, my IDE is configured to work with PSR Standards, I'll change the configuration to adapt to the WordPress standards and update the code.